sdk/nodejs/tests/provider/experimental/analyzer.spec.ts

```diff
@@ -340,12 +340,30 @@ describe("Analyzer", function () {
                         $ref: "/tls/v4.11.1/schema.json#/resources/tls:index%2FprivateKey:PrivateKey",
                         optional: true,
                     },
+                    outputParameterized: {
+                        $ref: "/terraform-provider/v0.2.2/schema.json#/resources/netlify:index%2FdeployKey:DeployKey",
+                    },
                 },
             },
         });
-        assert.deepStrictEqual(packageReferences, {
-            tls: "4.11.1",
-        });
+        assert.deepStrictEqual(
+            new Set(dependencies),
+            new Set([
+                {
+                    name: "tls",
+                    version: "4.11.1",
+                },
+                {
+                    name: "terraform-provider",
+                    version: "0.2.2",
```

Comment:
Isn't this the version of the base provider and should be `0.10.0`?

Reply:
Good catch, thanks!

Fixed here https://github.com/pulumi/pulumi/pull/19346/commits/77753aced2a1672ca6e684116d347aff8b2b92df